README.md

```diff
@@ -162,3 +162,23 @@ You can run automated tests, code analysis and Django configuration checks with:
 ```
 
 Always run this command before submitting code in a pull request and make sure that there are no warnings or failed tests.
+
+
+### Running MiddleMan
+
+You can run MiddleMan service with default values with:
+
+``` bash
+concent_api/manage.py middleman
+```
+
+You may add `-h` to check how to define non-default values.
+
+
+### Running Celery workers in development
+
+You can run Celery workers for Concent with:
```

Comment:
Please drop a few sentences about the workers. What they're for. Do you always have to run them? How many do you need? Do you need to have RabbitMQ running locally?

Maybe a quick command to run RabbitMQ in a docker container would be useful too:
``` bash
docker run \
    --rm \
    --hostname rabbitmq-server \
    --network host \
    --name rabbitmq \
    rabbitmq:alpine
```

Reply:
I would skip this one as for development it might be done in different (simpler) ways. For example, I am using `rabbitMQ` withou docker, and it is also possible to use `redis` for this. So I think for development it is not necessary.